twinkle-ansi/src/main/java/org/codejive/twinkle/ansi/Ansi.java

```diff
@@ -53,6 +53,15 @@ public class Ansi {
     public static final int COLORS_RGB = 2;
     public static final int COLORS_INDEXED = 5;
 
+    // OSC 8 hyperlinks (BEL-terminated)
```

Comment:
links are useful and osc-8 are commonly supported so i assume makes sense to add here?